src/hooks/useAxios.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react";
+import axios, { AxiosError, AxiosRequestConfig, AxiosResponse } from "axios";
+
+axios.defaults.baseURL = "https://devcademy.herokuapp.com/api";
```

Comment:
In projects you will want to keep this string in an environment variable and then set it from that variable here.

No need to do it here i'm just letting you know